api/services/pdf_auditor.py

```
@@ -0,0 +1,861 @@
+        """Extrai informações estruturais do PDF"""
+
+        # Contagem de objetos
+        total_objects = len(list(pdf_pikepdf.objects))
+
+        # Contagem de streams
+        total_streams = 0
+        for obj in pdf_pikepdf.objects:
+            try:
+                if hasattr(obj, 'read_bytes'):  # É um stream
+                    total_streams += 1
+            except:
```

Ruff (0.14.4)

Do not use bare `except`

(E722)